package.json

```diff
@@ -76,32 +83,47 @@
     "helmet": "3.8.2",
     "i18n-abide": "0.0.25",
     "joi": "10.4.1",
+    "jquery": "3.1.0",
+    "jquery-modal": "git://github.com/shane-tomlinson/jquery-modal.git#0576775d1b4590314b114386019f4c7421c77503",
+    "jquery-simulate": "1.0.2",
+    "jquery-ui": "1.12.1",
+    "jquery-ui-touch-punch-amd": "1.0.0",
+    "js-md5": "0.6.0",
     "jsxgettext-recursive": "git://github.com/vladikoff/jsxgettext-recursive#msgctxt-support",
+    "legal-docs": "git://github.com/mozilla/legal-docs.git#master",
     "load-grunt-tasks": "3.5.2",
     "lodash": "4.17.4",
+    "mailcheck": "git://github.com/mailcheck/mailcheck.git#d995fcc1d02c124a893c2cb0fce2e0ce5f36bce4",
     "mkdirp": "0.5.1",
     "mocha": "4.0.1",
     "morgan": "1.9.0",
     "mozlog": "2.2.0",
     "mustache": "2.3.0",
     "node-statsd": "0.1.1",
     "node-uap": "git://github.com/vladikoff/node-uap.git#9cdd16247",
+    "node-uuid": "1.4.8",
+    "node-vat": "0.0.9",
+    "normalize.css": "3.0.1",
     "on-headers": "1.0.1",
     "photon-colors": "1.0.3",
     "raven": "0.12.3",
+    "raven-js": "git://github.com/vladikoff/raven-js.git#customEndpoint-3.13.0",
     "serve-static": "1.13.1",
     "speed-trap": "git://github.com/rum-diary/speed-trap#0.0.7",
     "time-grunt": "1.4.0",
+    "ua-parser-js": "git://github.com/vladikoff/ua-parser-js.git#fxa-version",
     "uglifyjs-webpack-plugin": "1.1.8",
     "underscore": "1.8.3",
     "universal-analytics": "0.4.15",
     "webpack": "3.10.0",
-    "webpack-dev-middleware": "2.0.4"
+    "webpack-dev-middleware": "2.0.4",
```

Comment:
Should this be a dev dependency? Docs [recommend](https://github.com/webpack/webpack-dev-middleware) installing it as that.

Reply:
@vbudhram I shall check that!